Dockerfile

```diff
@@ -1,45 +1,7 @@
-# Build image
-FROM python:3.7-slim as builder
-
-# Setup virtualenv
-ENV VIRTUAL_ENV=/opt/venv
-RUN python -m venv $VIRTUAL_ENV
-ENV PATH="$VIRTUAL_ENV/bin:$PATH"
-
-# Install build deps
-RUN apt-get update
-RUN apt-get install -y --no-install-recommends build-essential pkg-config libpq-dev
-RUN python -m pip install --upgrade pip
-RUN pip install wheel
-
-# Install runtime deps
-COPY requirements.txt /tmp/requirements.txt
-RUN pip install -r /tmp/requirements.txt
-
-# Install c-lightning specific deps
-RUN pip install pylightning
-
-# Install LND specific deps
-RUN pip install lndgrpc
-
-# Production image
-FROM python:3.7-slim as lnbits
-
-# Run as non-root
-USER 1000:1000
-
-# Copy over virtualenv
-ENV VIRTUAL_ENV="/opt/venv"
-COPY --from=builder --chown=1000:1000 $VIRTUAL_ENV $VIRTUAL_ENV
-ENV PATH="$VIRTUAL_ENV/bin:$PATH"
-
-# Copy in app source
+FROM acaratti/pypoet:3.9
+COPY . .
```

Comment:
Will this copy also `docs`, `tests`, etc?

Reply:
they are on `.gitignore`